lib/process-manager.ts

```diff
@@ -370,6 +370,31 @@ export abstract class ProcessManager {
 
 		this.listen();
 	}
+	static exec(args: string, execOptions?: child_process.ExecOptions): Promise<string>;
+	static exec(args: [string, ...string[]], execOptions?: child_process. ExecFileOptions): Promise<string>;
+	static exec(args: string | string[], execOptions?: AnyObject) {
```

Comment:
tbh I think this would be better at the top level of ProcessManager than attached to QueryProcessManager.

Is there any reason you decided to put it on QueryProcessManager?

Reply:
It's not on QueryProcessManager? It's static on `ProcessManager`, but i assume since QueryProcessManager is a subclass, i can access the static from there.